Added new analog input read implementation

BenMendel opened the pull request:
Added a bunch of doc

BenMendel committed:
Added new analog input read implementation
Added a bunch of doc

BenMendel committed:
Writing a testing platform with e device detection

BenMendel committed:
Added "IsLineOpen" to the object replacing "is_relay_closed" method.
Configuring the testing program to work with the new objcect

BenMendel committed:
implementing DAQmxReadAnalogF64, DAQmxWriteAnalogScalarF64

BenMendel commented:
Hey @jrdnwrth! 
just checking if you got the chance to see my improvements on the project.

jrdnwrth commented:
Hi @BenMendel,

Sorry I missed this update!  I like what you have done.  There were a lot of changes.

You seem to understand the library better than I do.  Would you like to own the project?  If so, I would change my readme.md to point to your project so newcomers know where to go.

I don't know why it threw an AccessViolationException for you when trying to build as x64.  I'm on Windows 10 (22H2) 64 bit.  My nicaiu.dll is version 19.5.0.49152.  I was not able to reproduce this issue.

BenMendel commented:
Hi @jrdnwrth!

Thanks for the complements 😄 
Sure, I'd like to own this repository. Thank you!

jrdnwrth commented:
Hi @BenMendel 

I updated my repo to point to yours.  Enjoy!

Thank you.

BenMendel committed:
Add unit tests for NIDAQ functionality

- Updated solution to include "NetDaqmx.Tests" project.
- Added global using directive for Xunit in GlobalUsings.cs.
- Created NetDaqmx.Tests.csproj with necessary testing dependencies.
- Implemented NetDaqMxTests class with WritePort0Test and AnalogInputTest methods to validate NIDAQ operations.

jrdnwrth reviewed (approved):
Thank you!
